src/script/ptl-tool.py

```diff
@@ -42,51 +104,62 @@
         m = patt.match(line)
         CONTRIBUTORS[m.group(1)] = m.group(2)
 
-def build_branch(branch_name, pull_requests):
-    repo = git.Repo(".")
+def build_branch(args):
+    base = args.base
+    branch = args.branch
+    label = args.label
```

Comment:
@liewegas Could you please provide your inputs on [our discussion here](https://github.com/batrick/ceph/pull/4)  ? Thanks.

@liewegas @yuriw At the moment, the script validates the labels of type `wip-<name>-testing` and not `wip-<name>-testing2`. So would you like to update your labels to say `wip-sage2-testing`, to match the correct format ? (I think that is better). Otherwise, we have to incorporate that trailing number too. Please let me know.